frontend/tests/playwright/product/product_image.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { test, expect, Page } from '@playwright/test';
+
+test.describe('Product image test', () => {
+   test('product with a single image ', async ({ page }) => {
+      await page.goto('/products/iflex-opening-tool');
+
+      const image = page
+         .getByRole('img', { name: 'iFlex Opening Tool' })
+         .first();
+      expect(image).toBeVisible;
+   });
+
+   test('product with multiple images ', async ({ page }) => {
+      await page.goto('/products/repair-business-toolkit');
+
+      const firstImageSrc = await getCurrentActiveImgSrc(page);
+      const firstImage = page.locator(`img[src="${firstImageSrc}"]`).first();
```

Comment:
Are we sure this isn't grabbing the main active image? We want the image in the slider, right?

Reply:
> Are we sure this isn't grabbing the main active image? 

I am not exactly sure what you meant by the `main active image`. 

> We want the image in the slider, right?

Yes! We get the `src` of the image in the slider by `const firstImageSrc = await getCurrentActiveImgSrc(page)`.

Then use `const firstImage = page.locator(img[src="${firstImageSrc}"]).first()` to get the element, and use 
`await expect(firstImage).toBeVisible()` to make sure it's visible. 

Then we click on the next image and repeat the same process.

Finally, we compare the sources of the first and second images shown in the slider, and make sure 
they're different (as noted in https://github.com/iFixit/react-commerce/pull/1218#discussion_r1063006118).